app/carbonzipper/http_handlers.go

```diff
@@ -309,10 +309,11 @@ func (app *App) renderHandler(w http.ResponseWriter, req *http.Request) {
 	request.Trace.OutDuration = app.prometheusMetrics.RenderOutDurationExp
 	bs := app.filterBackendByTopLevelDomain(request.Targets)
 	bs = backend.Filter(bs, request.Targets)
-	metrics, points, mismatches, errs := backend.Renders(ctx, bs, request,
-		app.config.RenderMismatchCheck, app.config.RenderMismatchMetricReportLimit)
+	metrics, points, mismatches, fixedMismatches, errs := backend.Renders(ctx, bs, request,
```

Comment:
This has grown from 2 return value to 5 return values within a span of a few weeks. I wonder if there is place for some kind of struct for this data 🤔

Reply:
Yeah I totally agree. It's become ridiculous. I would go with something like RenderStats struct. Last time, team told me not to use struct and just return two values. So if you verify that this is a good idea, I'll make the change.

Reply:
my vote'd be for struct

Reply:
Made the change.

Reply:
Can you resolve this if you're ok with this now?